### PR TITLE
Adds dual samples for many sounds

### LINKED ISSUE #62: Dual Samples
Some of the included samples are terribly irritating. An example is the Bonfire sample. It's about 1 1/2 seconds long, and for all the world sounds like somebody left a record player on after the record was finished.

Sounds which by nature will have a repetitive pattern like this one are easily masked, however. You simply have two separate samples of different lengths. For example, for this bonfire, I would recommend two samples, one eight seconds long, and one 15 seconds long. If you start them both at the same time, and repeat them both (assuming a smooth transition on both samples, of course), it would be several minutes before the pattern repeats.

Both samples should be of equal volume, and could both be controlled by the same volume selector. This functionality is simply to provide a non repetative sample, it's not necessary to have it appear as two separate samples. ;)

### PR DESCRIPTION
**Pull request by ashutoshgngwr**
### Changes
- Sound declarations can now have multiple source samples to play simultaneously
- Make necessary changes to the Chromecast receiver applications as well
- Optimize graphics for Chromecast UI (should help with OOM errors on speaker devices [just a guess though])
- Split existing relevant sounds into two samples (key result: reduction in size on disk by 42%)

### Testing
- [x] Tested on a physical device
- [x] Added or modified unit test cases

### Others
- Fixes #62 

### TODO

- [x] Add a different icon for casting status on cast-receiver
- [x] Fix app icon in cast receiver (icon in fastlane metadata is not transparent anymore)
- [x] Rename `PlayerManagerStatusEvent` to `PlayerManagerStatus`

### CODE REVIEW

**Comment by ashutoshgngwr** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 13
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ashutoshgngwr/noice/pullRequest?prid=6042010&bid=19797220)

File: cast-receiver/src/player_manager.ts [hunk 1]
```diff
@@ -128,93 +128,94 @@ export default class PlayerManager {
...
-      if (event.action === undefined || event.action === null) {
-        return;
-      }
-
-      if (event.action !== PlayerAction.Create) {
-        // shouldn't be here?
-        return;
-      }
+  handlePlayerEvent(event: PlayerControlEvent): void {
+    event.src.forEach((soundKey: string) => {
```
Comment (ashutoshgngwr) [inline cast-receiver/src/player_manager.ts:160]: Codacy found an issue: [Arrow function has a complexity of 10. Maximum allowed is 4.](https://app.codacy.com/manual/ashutoshgngwr/noice/pullRequest?prid=6042010)

File: cast-receiver/src/status_ui_handler.ts [hunk 2]
```diff
@@ -1,104 +1,102 @@
...
-   * showing ready to cast message, the method does nothing.
+   * Enables the status with given ID.
+   *
+   * @param statusID ID of status to toggle. must be one of
+   * StatusUIHandler.IDLE_STATUS_ID or StatusUIHandler.CASTING_STATUS_ID
+   * @param enabled whether to display or hide the given status
    */
-  showIdleStatus(): void {
-    if (this.isShowingIdleStatus() === true) {
+  enableStatus(statusID: string, enabled: boolean): void {
```
Comment (ashutoshgngwr) [inline cast-receiver/src/status_ui_handler.ts:78]: Codacy found an issue: [Method 'enableStatus' has a complexity of 7. Maximum allowed is 4.](https://app.codacy.com/manual/ashutoshgngwr/noice/pullRequest?prid=6042010)

File: cast-receiver/src/status_ui_handler.ts [hunk 3]
```diff
@@ -1,104 +1,102 @@
...
+        StatusUIHandler.LOADER_STATUS_ICON,
+        ""
+      )
+    );
   }
 
   private createIcon(id: string): HTMLElement {
     const template = document.createElement("template");
-    template.innerHTML = `<svg id="${id}" class="sound"><use xlink:href="${Icons[id]}" /><svg>`;
+    template.innerHTML = `<svg id="${id}" class="icon"><use xlink:href="${Icons[id]}" /><svg>`;
```
Comment (ashutoshgngwr) [inline cast-receiver/src/status_ui_handler.ts:54]: Codacy found an issue: [Unsafe assignment to innerHTML](https://app.codacy.com/manual/ashutoshgngwr/noice/pullRequest?prid=6042010)

Comment (ashutoshgngwr) [inline cast-receiver/src/status_ui_handler.ts:54]: Codacy found an issue: [Generic Object Injection Sink](https://app.codacy.com/manual/ashutoshgngwr/noice/pullRequest?prid=6042010)

**Comment by ashutoshgngwr** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 13
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ashutoshgngwr/noice/pullRequest?prid=6042010&bid=19797220)

File: cast-receiver/src/player_manager.ts [hunk 4]
```diff
@@ -128,93 +124,94 @@ export default class PlayerManager {
...
```
Comment (ashutoshgngwr) [inline cast-receiver/src/player_manager.ts:156]: Codacy found an issue: [Arrow function has a complexity of 10. Maximum allowed is 4.](https://app.codacy.com/manual/ashutoshgngwr/noice/pullRequest?prid=6042010)

File: cast-receiver/src/status_ui_handler.ts [hunk 5]
```diff
@@ -1,104 +1,102 @@
...
-   * showing ready to cast message, the method does nothing.
+   * Enables the status with given ID.
+   *
+   * @param statusID ID of status to toggle. must be one of
+   * StatusUIHandler.IDLE_STATUS_ID or StatusUIHandler.CASTING_STATUS_ID
+   * @param enabled whether to display or hide the given status
    */
-  showIdleStatus(): void {
-    if (this.isShowingIdleStatus() === true) {
+  enableStatus(statusID: string, enabled: boolean): void {
```
Comment (ashutoshgngwr) [inline cast-receiver/src/status_ui_handler.ts:78]: Codacy found an issue: [Method 'enableStatus' has a complexity of 7. Maximum allowed is 4.](https://app.codacy.com/manual/ashutoshgngwr/noice/pullRequest?prid=6042010)

File: cast-receiver/src/status_ui_handler.ts [hunk 6]
```diff
@@ -1,104 +1,102 @@
...
+        StatusUIHandler.LOADER_STATUS_ICON,
+        ""
+      )
+    );
   }
 
   private createIcon(id: string): HTMLElement {
     const template = document.createElement("template");
-    template.innerHTML = `<svg id="${id}" class="sound"><use xlink:href="${Icons[id]}" /><svg>`;
+    template.innerHTML = `<svg id="${id}" class="icon"><use xlink:href="${Icons[id]}" /><svg>`;
```
Comment (ashutoshgngwr) [inline cast-receiver/src/status_ui_handler.ts:54]: Codacy found an issue: [Unsafe assignment to innerHTML](https://app.codacy.com/manual/ashutoshgngwr/noice/pullRequest?prid=6042010)

Comment (ashutoshgngwr) [inline cast-receiver/src/status_ui_handler.ts:54]: Codacy found an issue: [Generic Object Injection Sink](https://app.codacy.com/manual/ashutoshgngwr/noice/pullRequest?prid=6042010)

**Comment by Lesik** [comment]
Hi @ashutoshgngwr,

first off, thanks a lot for this great app!

I noticed that this MR modified `app/src/main/assets/airplane_seatbelt_beeps.mp3`, I assume related to the "Split existing relevant sounds into two samples". Before the change, the sound consisted of two beeps, high-middle. After the change, it is three beeps low-high-middle.

I was wondering if this change was deliberate, as according to my understanding this MR should only split effects into separate files to playback intertwinedly, not change the content of the sounds. In case it was an accident, I would be very happy if you could revert the change to this particular effect, as it was one of my favorite effects in the library and in my personal opinion the third tone quite ruins it.

Before: https://raw.githubusercontent.com/ashutoshgngwr/noice/7f4d5fa97e460411924bc2b4b5422a776fbde06d/app/src/main/assets/airplane_seatbelt_beeps.mp3

After: https://raw.githubusercontent.com/ashutoshgngwr/noice/d6660000c2bac06928ef6730442ed45466fddbdf/app/src/main/assets/airplane_seatbelt_beeps.mp3

Thanks!

**Comment by ashutoshgngwr** [comment]
@Lesik it was a deliberate change. Long story short, original sample had 3 beeps (current version) but I trimmed one beep in earlier versions (personal preference). Don't remember why I changed it though. Feel free to submit a PR reverting to old sample. :)

**Comment by Lesik** [comment]
@ashutoshgngwr I was going to write "if you could accept a MR" but then I realized that many samples (including the seatbelt beeps) have been equalized since (#313). Just reverting the original file would result in an unequalized sample, so it's not that easy for me to submit a MR. Could you document the process you used for equalization?

**Comment by ashutoshgngwr** [comment]
@Lesik https://github.com/ashutoshgngwr/noice/issues/227#issuecomment-685164223

**Pull request merged**